Blog post for the 3 September call (#108)
publiccodenet/blog: _posts/2020-09-22-notes-from-community-call-3-september-2020.md

---
title: "Notes from community call - 3 September 2020"
date: 2020-09-22
author: Jan Ainali
type: blogpost

> excerpt: Discussions about processes for how to do translations of the Standard for Public Code

Reviewer: ```suggestion
excerpt: Discussion about processes to translate the Standard for Public Code
```

* Next week we will start our live stream / podcast [Let's talk about public code](https://blog.publiccode.net/news/2020/09/01/on-air-next-our-podcast.html).
* Boris will present on why 'open source in gov't is different' at State of the Source on 9 September ([video](https://www.youtube.com/watch?v=mvhFF_7lVV4&list=PL_5ziu2gADmDcp_ER8x2rcT1HOBaIe5cl&index=3)).

And for more news, you can always head to our [blog](https://blog.publiccode.net/).

## Attendees

* [Alba Roza](https://publiccode.net/team/alba-roza.html) (Chair)
* [Jan Ainali](https://publiccode.net/team/jan-ainali.html)
* [Eric Herman](https://publiccode.net/team/eric-herman.html)
* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
* Marc van Andel
* Martin Dias d'Ullios
* Matti Schneider
* Olaf-Gerd Gemein

## Notes

National public organizations might have access to professional translators in the EU languages which opens up a totally different path than traditional community translations by volunteers.

> The Foundation for Public Code could then be the one that take institutional responsibility for overseeing and merging translations in collaboration with a community for each language.

Reviewer: ```suggestion
The Foundation for Public Code could then take institutional responsibility for overseeing and merging translations in collaboration with a community for each language.
```

> This solution seemed particulary interesting and the Foundation for Public Code will look into this path next.

Reviewer: ```suggestion
This solution seemed particularly interesting and the Foundation for Public Code will look into this path next.
```

As a short term solution the Standard for Public Code could also be modularized further.
Of particular interest would be an executive summary that also would be quicker to translate and at the same time might be the most important part to localize.

> During the meeting a summary of the previous call and how the Foundation for Pulic Code have been thinking about translations was given.

Reviewer: ```suggestion
Foundation for Public code staff gave a [summary of the previous community call about translations](https://blog.publiccode.net/community%20call/2020/08/24/notes-from-community-call-6-august-2020.html) and how we've been thinking about translations.
```

> Foundation for Public code staff gave a [summary of the previous community call about translations](https://blog.publiccode.net/community%20call/2020/08/24/notes-from-community-call-6-august-2020.html) and how we've been thinking about translations.

Reviewer: ```suggestion
Foundation for Public code staff gave a summary of the [previous community call about translations](https://blog.publiccode.net/community%20call/2020/08/24/notes-from-community-call-6-august-2020.html) and how we've been thinking about translations.
```

> Jan described how it will be a lot easier for people just doing translations if platforms purposed for this are used (having translation memory, documentation about what's being translated etc.) and how those can be automated to commit to a git repository.

Reviewer: ```suggestion
Jan described how it would be a lot easier for people just doing translations if platforms designed for this are used (with features like translation memory or documentation about what's being translated) and how those can be automated to commit to a git repository.
```

Boris described another system with forks with version numbers.

> This can be enabled by having one leading version which lets the Standard for Public Code to evolve in its own pace.

Reviewer: ```suggestion
This can be enabled by having one leading version which lets the Standard for Public Code to evolve at its own pace.
```

> Each fork could then choose any system they wanted for translation that suits their language community.

Reviewer: ```suggestion
Each fork could then choose any system they wanted for translation to suit their language community.
```
